### PR TITLE
feat(cost-analysis-l-n-b-store): create cost-analysis-l-n-b-store

### PR DESCRIPTION
**Pull request by piggggggggy**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [x] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA

- Create `cost-analysis-l-n-b-store`. Add Data-Source state
- Separate Cost-Query logic from `cost-analysis-page-store`. (to `cost-analysis-l-n-b-store`)

### CODE REVIEW

File: apps/web/src/services/cost-explorer/store/cost-analysis-l-n-b-store.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+}
+
+export const useCostAnalysisLNBStore = defineStore('cost-analysis-l-n-b', {
+    state: (): CostAnalysisLNBState => ({
+        costQueryList: [],
+        dataSourceList: [
+            'aws',
+            'azure',
+            'gcp',
+        ],
```
Comment (WANZARGEN) [inline apps/web/src/services/cost-explorer/store/cost-analysis-l-n-b-store.ts]: nitpick: Please add TODO comment not to miss updating it and to notify reviewers that this is not the mistake. :)

Comment (yuda110) [inline apps/web/src/services/cost-explorer/store/cost-analysis-l-n-b-store.ts:25]: question: this is mock data, right?

File: apps/web/src/services/cost-explorer/store/cost-analysis-page-store.ts [hunk 2]
```diff
@@ -178,15 +158,15 @@ export const useCostAnalysisPageStore = defineStore('cost-analysis-page', {
                     name,
                     options,
                 });
-                this.selectedQueryId = createdData.cost_query_set_id;
+                costAnalysisLNBStore.$patch({ selectedQueryId: createdData.cost_query_set_id });
```
Comment (WANZARGEN) [inline apps/web/src/services/cost-explorer/store/cost-analysis-page-store.ts:161]: praise: Oh, I didn't know that we can use pinia store inside the other pinia store. 😮
Thank you for introducing me to a new way of using it.

File: apps/web/src/services/cost-explorer/store/cost-analysis-l-n-b-store.ts (hunk 1 above)
Comment (piggggggggy) [reply apps/web/src/services/cost-explorer/store/cost-analysis-l-n-b-store.ts:25]: oh.. yes... I forgot that.. I'll remove them.

**Review by WANZARGEN — APPROVED** [review]
LGTM~ Nice refactoring!
It seems that the $reset() and $dispose() functions of this store are not being called at all in the code. It might be worth considering whether these functions should be executed during the creation or unmounting of the LNB, if that hasn't been reviewed already.

**Pull request merged**